File: ts-sdk/src/sdk/models/operations/getadeployrequest.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetADeployRequestPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}


export class GetADeployRequestRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetADeployRequestPathParams;
}


export class GetADeployRequestResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getADeployRequest200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: go-sdk/pkg/models/operations/getadeployrequest.go
package operations

type GetADeployRequestPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type GetADeployRequestRequest struct {
	PathParams GetADeployRequestPathParams
}

type GetADeployRequestResponse struct {
	ContentType                               string
	StatusCode                                int
	GetADeployRequest200ApplicationJSONObject map[string]map[string]interface{}
}
